test/test.ts

```diff
@@ -170,102 +158,31 @@ Deno.test("sqlite", async (t) => {
   });
 
   await t.step("query with string param", () => {
-    const row = db.queryArray<[number, string, number, Uint8Array, null]>(
+    const row = db.query(
       "select * from test where text = ?",
-      "hello world",
-    )[0];
+    ).values<[number, string, number, Uint8Array, null]>("hello 0")[0];
+
     assertEquals(row[0], 0);
-    assertEquals(row[1], "hello world");
+    assertEquals(row[1], "hello 0");
     assertEquals(row[2], 3.14);
-    assertEquals(row[3], new Uint8Array([1, 2, 3]));
+    assertEquals(row[3], new Uint8Array([3, 2, 1]));
     assertEquals(row[4], null);
   });
 
-  await t.step("query with string param (template string)", () => {
-    const row = db.queryArray<
-      [number, string, number, Uint8Array, null]
-    >`select * from test where text = ${"hello world"}`[0];
+  await t.step("query with string param (named)", () => {
+    const row = db.query(
+      "select * from test where text = :p1",
+    ).values<[number, string, number, Uint8Array, null]>({ p1: "hello 0" })[0];
+
     assertEquals(row[0], 0);
-    assertEquals(row[1], "hello world");
+    assertEquals(row[1], "hello 0");
     assertEquals(row[2], 3.14);
-    assertEquals(row[3], new Uint8Array([1, 2, 3]));
+    assertEquals(row[3], new Uint8Array([3, 2, 1]));
     assertEquals(row[4], null);
   });
 
-  await t.step("more than 32-bit int", () => {
```

Comment:
Need these tests back & to pass again

Reply:
@littledivy we need to support 64-bit integers correctly, but that can't be done without affecting performance by quite a lot. How should we determine if column of type integer (which is quite ambiguous) is 64-bit or 32-bit to use appropriate function? If we just use sqlite3_column_int64 first and then convert to number if its 32 bit that is going to have quit an impact on performance. Or could we have user specify that a column must be retrieved specifically as an BigInt/int64 using a new API?

Reply:
Just retrieving int64s by default for now. We can work out more performant solution later